feat: at file in chat input box

abeatrix opened the pull request:
CLOSE https://github.com/sourcegraph/cody/issues/1523

## WIP

![image](https://github.com/sourcegraph/cody/assets/68532117/4babf75d-6c7d-44ef-b09c-c5f880602597)


## Test plan

<!-- Required. See https://docs.sourcegraph.com/dev/background-information/testing_principles. -->

toolmantim commented:
Woohoo! I know this is a WIP, but I just gave it a go to give some early feedback. Awesome start. Two things I noticed:
1. The menu doesn't seem to trigger consistently when you type @… seems to want you to type a forward slash or . before appearing
2. Hitting Enter when the menu is showing shouldn't submit, that should only act as a selection

toolmantim commented on vscode/webviews/ChatFileSelector.tsx:
```diff
@@ -0,0 +1,69 @@
+
+    return (
+        <div className={classNames(styles.container)}>
+            <div className={classNames(styles.headingContainer)}>
+                <h3 className={styles.heading}>Add selected file as context...</h3>
+            </div>
```
Let's remove this heading to start with

abeatrix commented:
> Woohoo! I know this is a WIP, but I just gave it a go to give some early feedback. Awesome start. Two things I noticed:
> 
> 1. The menu doesn't seem to trigger consistently when you type @… seems to want you to type a forward slash or . before appearing
> 2. Hitting Enter when the menu is showing shouldn't submit, that should only act as a selection

Thanks for testing this out! I've fixed the both issues  in the PR.  Also fixed the same issue (#1) on main in https://github.com/sourcegraph/cody/pull/1592
